### PR TITLE
Use unified Docker image.

### PR DESCRIPTION
**Pull request by epgts**

### CODE REVIEW

File: tools/testbin [hunk 1]
```diff
@@ -32,12 +32,17 @@ set -ex
 MIN_DEB_ARM=1.10.1
 # We added 1: epoch at 1.7.0.
 MIN_DEB_EPOCH=1.7.0
-# TODO Drop default and require -pgversions after existing usage updated.
-PG_VERSIONS='12 13 14'
+# TODO Unfortunate that pgx allows neither specifying nor querying the port it
+#   starts postgres on, so we duplicate that knowledge.  Watch out for
+#   that changing!
+PGX_PORT_BASE=28800
```
Comment (Smittyvb) [inline tools/testbin:38]: pgx will support specifying this in the next releases: https://github.com/tcdi/pgx/pull/765

Comment (rtwalker) [reply tools/testbin:38]: @epgts since you've merged #589 you might try to use the new port stuff

Comment (epgts) [reply tools/testbin:38]: Thanks, Smitty and Ryan.  I'll have a look at that at some point.  It's not a slam dunk as currently implemented, as they only accept the base port in the configuration, not as a parameter to the start&stop commands.

Not urgent, anyway.

**Review by WireBaron — APPROVED** [review]
This looks good

**Pull request merged**